frontend-react/e2e/spec/daily-data-page.spec.ts

```diff
@@ -93,11 +101,14 @@ test.describe("Daily Data page", () => {
         });
 
         test("table has correct headers", async ({ page }) => {
-            await expect(page.getByText(/Report ID/)).toBeAttached();
-            await expect(page.getByText(/Time received/)).toBeAttached();
-            await expect(page.getByText(/File available until/)).toBeAttached();
-            await expect(page.getByText(/Items/)).toBeAttached();
-            await expect(page.getByText(/Filename/)).toBeAttached();
+            const table = page.locator("table");
+            await expect(table.getByText(/Report ID/)).toBeAttached();
```

Comment:
Can we update this to
await expect(page.locator(".usa-table th").nth(0)).toHaveText(
                    /Report ID/,
                );